frontend/src/firebase/firebase.js

```diff
@@ -0,0 +1,21 @@
+import firebaseConfig from './firebase.json'
+import {
+  getAuth,
+  signOut,
+  GoogleAuthProvider,
+  signInWithPopup,
+} from 'firebase/auth'
+import { initializeApp } from 'firebase/app'
+
+initializeApp(firebaseConfig)
+export const auth = getAuth()
+
+// function attempting to sign in with Google
+export async function signInSSO() {
+  const provider = new GoogleAuthProvider()
+  const userCredential = await signInWithPopup(auth, provider)
+}
```

Comment:
I suppose the function name can be changed because it's not SSO anymore. Also unused variable - userCredential?

Reply:
I left it as a variable assignment because for some reason, if I didn't assign it to a variable, it wouldn't work. I changed it again but left the await and it works now.